.github/workflows/ci.yml

```diff
@@ -16,16 +16,19 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - uses: dtolnay/rust-toolchain@1.70
+      - uses: dtolnay/rust-toolchain@master # avoid the tag here to prevent dependabot from updating it
```

Comment:
this might be a bad idea if the base action has a breaking change across major versions (which it's allowed to do).

in the year 2024 it would probably be worth just switching to [actions-rust-lang/setup-rust-toolchain](https://github.com/actions-rust-lang/setup-rust-toolchain) which actually has a normal versioning scheme (i.e. it doesn't bump the major version of every rust update).

Reply:
I've switched it to [actions-rust-lang/setup-rust-toolchain](https://github.com/actions-rust-lang/setup-rust-toolchain).

Reply:
@refi64 do you have comparison between both actions apart from your matter-of-fact type statement? I'm not familiar with actions-rust-lang so i'm not sure what switching brings us? It does seem less popular, but that's not necessarily a good indication.

fwiw  dtolnay/rust-toolchain@master pattern is specifically documented by that action. And i'm not a fan of a single crate doing a different action then all the others :p (ofcourse if there is a real rationale, we can swtich them all)

Reply:
@ColinKinloch can you roll that back to not change the action used, the nwe can merge it and punt the conversation about which action we should move to in 2025 if any